### DISCUSSION TITLE
Cron for RFID and GPIO

### DISCUSSION THREAD

**Discussion by UbiGIT89** [MiczFlor/RPi-Jukebox-RFID#1539] [Q&A]
Hi Community,

My girls using her boxes until late night, so My question is:

Can I a cron Job which deactivate GPIO and RFID at 7pm, so they cannot Start something new, but can hear the activ list until its finished.
Of course in the morning it should be activated.

Best regards
Torben

**Comment by pabera** [comment]
What an interesting parent problem I can resonate a lot with :-) And a peculiar solution indeed.

I am not super familiar with the code of V2.x but I did a [quick search on `Enable GPIO`](https://github.com/MiczFlor/RPi-Jukebox-RFID/search?q=Enable+GPIO) (it's what you see in the Webapp in the settings page)

In the search results, you see a few hints you might be able to work with like ...

* `sudo systemctl stop phoniebox-gpio-control.service` (I think this is the one you need)
* `sudo systemctl stop phoniebox-gpio-buttons`

Similarly, there is something related to [`rfid service`](https://github.com/MiczFlor/RPi-Jukebox-RFID/search?q=rfid+service) as well.

* sudo systemctl stop phoniebox-rfid-reader.service

By using `sudo systemctl stop`, the service will be started again next time the Phoniebox reboots. You could bypass it by restarting the box, though. If your daughters figure that out, you might need to be a little smarter and add an autostart service that checks the current time and immediately disables the services after boot.

Good luck!
